jalien-root.sh

```diff
@@ -1,9 +1,13 @@
 package: JAliEn-ROOT
 version: "%(tag_basename)s"
-tag: "0.1.0"
-source: https://gitlab.cern.ch/jalien/jalien-root
+tag: "0.2.0"
+source: https://gitlab.cern.ch/jalien/jalien-root.git
 requires:
   - ROOT
+env:
```

Comment:
This is not effective: `env:` does not make envvars available in this recipe at build time. It makes, however, those variables available when building all the recipes that depend on it, incorrectly overriding the main defaults. Removing this section. I am taking care of it.